.cicd/release.sh

```diff
@@ -15,4 +15,14 @@ set -eou pipefail
   helm package .
 
   mv "weaviate-$VERSION.tgz" "weaviate.tgz"
+
+  echo "Add chart version $VERSION to GitHub Pages"
+  cd ..
+  mkdir -p files-to-gh-pages
```

Comment:
you can define a variable with `files-to-gh-pages` and then you can just reference it through the script

Reply:
I think that also you can add `files-to-gh-pages` it `.gitignore`

Reply:
`files-to-gh-page` does not exist, so I guess if I use a ENV VAR I would have to check if the directory exists. The `-p` option in `mkdir` is going to create a directory only if it does not exist.

Why should we add `files-to-gh-page` to `.gitignore`? The folder does not exist so there is nothing to ignore, right?

Reply:
but if you run `release.sh` script locally, then the directory gets created and that's why I was thinking of maybe it should be in `.gitignore` so that after that operation someone won't commit this accidentally, but this is a really rare scenario, so we can skip it

Reply:
I was actually referring to this:
```
GH_PAGES_DIR="files-to-gh-pages"
mkdir -p $GH_PAGES_DIR
```
so that we would have the directory name defined once using a variable